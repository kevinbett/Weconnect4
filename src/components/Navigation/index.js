import React from 'react';
import {Link} from 'react-router-dom';

const Navigation = () => (

  <div className="navbar navbar-expand-lg navbar-light bg-light">
        <Link className="navbar-brand" to="/">Home</Link>
        <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
          aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
          <span className="navbar-toggler-icon"></span>
        </button>

        <div className="collapse navbar-collapse" id="navbarSupportedContent">
          <ul className="navbar-nav mr-auto">
            <li className="nav-item active">
              <Link className="nav-link" to="/registerbusiness">Register Business <span className="sr-only">(current)</span></Link>
            </li>
            <li className="nav-item">
              <Link className="nav-link" to="/viewbusiness">View Businesses</Link>
            </li>
            <li className="nav-item">
              <Link className="nav-link" to="/logout">LogOut</Link>
            </li>
            <li className="nav-item">
              <Link className="nav-link" to="/Search">Search</Link>
            </li>
          </ul>
          <form className="form-inline my-2 my-lg-0">
            <input className="form-control mr-sm-2" type="" placeholder="Search" aria-label="Search" />
            <button className="btn btn-outline-success my-2 my-sm-0" type="">Search</button>
          </form>
          
        </div>
      </div>

);

export default Navigation;