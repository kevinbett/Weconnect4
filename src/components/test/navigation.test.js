import Navigation from "../Navigation"
import { shallow } from "enzyme"
import { Divider } from "../../../node_modules/semantic-ui-react";

describe('Navigation bar',() => { 
    it ('renders navigation bar components', () => { 
        expect(shallow(<Navigation/>)).toMatchSnapshot()
    });
} )
