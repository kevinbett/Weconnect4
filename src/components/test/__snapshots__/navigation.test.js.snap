// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation bar renders navigation bar components 1`] = `
<div
  className="navbar navbar-expand-lg navbar-light bg-light"
>
  <Link
    className="navbar-brand"
    replace={false}
    to="/"
  >
    Home
  </Link>
  <button
    aria-controls="navbarSupportedContent"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarSupportedContent"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarSupportedContent"
  >
    <ul
      className="navbar-nav"
    >
      <li
        className="nav-item"
      >
        <Link
          className="nav-link"
          replace={false}
          to="/viewbusiness"
        >
          View Businesses
        </Link>
      </li>
    </ul>
    <div />
  </div>
</div>
`;
